src/routes/auth.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { AuthController } from '../controllers/AuthController';
+
+const router = express.Router();
+const authController = new AuthController();
+
+router.post('/login', authController.login);
+router.post('/register', authController.register);
```

Comment:
we should consider adding a logout route as well so we can support a full flow when testing. meaning a logout will reset the cache etc